### ISSUE TITLE
API Explorer - Fields repeated for certain records

### ISSUE THREAD

**Issue by SimmoK** [NationalMuseumAustralia/Collection-API#83]
![apiexplorerbug](https://user-images.githubusercontent.com/36829954/44122712-6e63721e-a068-11e8-954e-60433b1a65fd.JPG)
I've noticed that sometimes the medium or related spatial/temporal/people are repeated multiple times. See example attached. I am using Chrome to view  

Simon

**Comment by staplegun** [comment]
It is a bug in the API data mapping.

EMu XML:
```xml
 <record>
  <irn>54041</irn>
  <PhyMaterials_tab>
   <PhyMaterial>Textile - non specific</PhyMaterial>
   <PhyMaterial>Leather</PhyMaterial>
  </PhyMaterials_tab>
```

JSON-LD
```javascript
made_of: [
{
  type: "Material",
  label: "Textile - non specific",
  see_also: "http://data.nma.gov.au/object?medium=%22Textile%20-%20non%20specific%22"
},
{
  type: "Material",
  label: "Leather",
  see_also: "http://data.nma.gov.au/object?medium=%22Leather%22"
},
{
  type: "Material",
  label: "Leather",
  see_also: "http://data.nma.gov.au/object?medium=%22Leather%22"
},
{
  type: "Material",
  label: "Textile - non specific",
  see_also: "http://data.nma.gov.au/object?medium=%22Textile%20-%20non%20specific%22"
}
],
```

**Comment by SimmoK** [comment]
Appears to affect the 'Additional Type' field as well

**Comment by SimmoK** [comment]
Measurement unit is also doubled up. Not sure if this is the same bug
![unti](https://user-images.githubusercontent.com/36829954/44177275-849a3180-a130-11e8-8fec-dc66e72735d2.JPG)

**Comment by Conal-Tuohy** [comment]
record is OK in nma-dev after switching to TDB1:
http://nma-dev.conaltuohy.com/object/54041#

**Comment by Conal-Tuohy** [comment]
Thanks Fiona! 
I had run an incremental ETL on nma-dev last night, and that record had not updated because the XML source data for it had not changed. So I deleted the record from Solr (just now), and ran the incremental update again. As you can see http://nma-dev.conaltuohy.com/object/65064 is fixed now. So running the ETL in "full" mode will over-write the entire set of Solr records and should fix them all.

**Comment by Conal-Tuohy** [comment]
Fixed in https://github.com/NationalMuseumAustralia/Collection-API-ETL/commit/2b15b77abbdbd683eff632a32bf445207d823d80

**Comment by f27wood** [comment]
Foxed for those records already updated, waiting for full ETL load to test for all records.

**Comment by f27wood** [comment]
It look sliek most of the records are AOK now.. I did come across one where the AdditionalType field had 3 duplicate values:

https://data.nma.gov.au/apiexplorer.html?id=object/47157

Is there a way to test if any other records have duplicate data?

**Comment by f27wood** [comment]
Hmm I just checked that record in EMu, and it doesn't exist! Something odd going on there...

**Comment by Conal-Tuohy** [comment]
Shouldn't this have been given the status `Removed`?

This record would be a great candidate for testing the tracking of deleted records! #40

**Comment by SimmoK** [comment]
Yes my thoughts exactly, I'll ask Karen. I check the export scripts and the do filter on record status. I'm not sure how this record turned up. I might have to test the add/or clauses in the query.

**Comment by SimmoK** [comment]
Ah this record was marked as 'retired' on 28/8/2018 and doesn't show up in the later full which means i am filtering correctly but Registration must add a check of the API field as part of their retirement process.

**Comment by f27wood** [comment]
Simon and I have discussed this and from now on, he will include all records that have an API status including those that are not active. And Registration will change the API status to "Removed" for any retired records.

This means if an active record was in the API, and was then retired, it will be included in the XML with the API status of "Removed" so you know to remove it.
